extension/test/unit/payment-handler.spec.js

```diff
@@ -209,7 +38,7 @@ describe('payment-handler::execute', () => {
 
   describe('amountPlanned', () => {
     it(
-      'is different than the amount in makePayment request custom field and interface interaction is empty, ' +
+      'is different than the amount in create session request custom field and interface interaction is empty, ' +
```

Comment:
Do we need to change the implementation of this test case? I found it is still accessing  `ctpPaymentClone.custom.fields.makePaymentRequest`

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/1050/commits/ee56b5d6d6d7cd8a46d4cdd9a801f972df458681

Reply:
Aren't we going to use create session to test the `amountPlanned` here? I think we don't use `makePayment` thingy anymore in this test case.

Reply:
Follow up ticket created to handle removal of makePayment request handler.
https://jira.commercetools.com/browse/CTPI-761

And this test is specific to makePayment.